.github/workflows/check-clog-linux.yml

```diff
@@ -0,0 +1,31 @@
+name: CLOG
```

Comment:
I recommend having unique names for all these.

Reply:
agreed; fixed